### PR TITLE
TF-M:fix build error withouth gpio0 enabled

### PR DESCRIPTION
**Pull request by joerchan**

### CODE REVIEW

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 1]
```diff
@@ -3823,6 +3823,19 @@ Trusted Firmware-M (TF-M)
 
 The issues in this section are related to the TF-M implementation in the |NCS|.
 
+.. rst-class:: v2-5-0
+
+NCSDK-24804: TF-M does not withouth gpio0 node enabled in devicetree
```
Comment (SebastianBoe) [inline doc/nrf/releases_and_maturity/known_issues.rst:3828]:
```suggestion
NCSDK-24804: TF-M does not build when the gpio0 node is disabled in devicetree
```

**Review by SebastianBoe — APPROVED** [review]

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 2]
```diff
@@ -3823,6 +3823,19 @@ Trusted Firmware-M (TF-M)
...
+  This fails with error: 'TFM_PERIPHERAL_GPIO0_PIN_MASK_SECURE' undeclared.
```
Comment (mia-ko) [inline doc/nrf/releases_and_maturity/known_issues.rst:3829]:
```suggestion
  This fails with the error message ``'TFM_PERIPHERAL_GPIO0_PIN_MASK_SECURE' undeclared``.
```

File: doc/nrf/releases_and_maturity/known_issues.rst [hunk 3]
```diff
@@ -3823,6 +3823,19 @@ Trusted Firmware-M (TF-M)
...
+
+  **Workaround:** Enable gpio0 node in devicetree configuration
```
Comment (mia-ko) [inline doc/nrf/releases_and_maturity/known_issues.rst:3831]:
```suggestion
  **Workaround:** Enable the GPIO0 node in the devicetree configuration.
```

File: doc/nrf/releases_and_maturity/known_issues.rst (hunk 1 above)
Comment (mia-ko) [reply doc/nrf/releases_and_maturity/known_issues.rst:3828]: Also, should we write GPIO0?

**Review by mia-ko — CHANGES_REQUESTED** [review]

**Review by Vge0rge — APPROVED** [review]

File: doc/nrf/releases_and_maturity/known_issues.rst (hunk 1 above)
Comment (joerchan) [reply doc/nrf/releases_and_maturity/known_issues.rst:3828]: I was using lower-case because I'm referring to the node in devicetree, which is lower-case.

Comment (mia-ko) [reply doc/nrf/releases_and_maturity/known_issues.rst:3828]: I just noticed that we are writing GPIO0 and GPIO1 in Kconfig file as well, so I thought we should align :)

File: doc/nrf/releases_and_maturity/known_issues.rst (hunk 3 above)
Comment (joerchan) [reply doc/nrf/releases_and_maturity/known_issues.rst:3831]: @mia-ko Looking into how we have done this elsewhere, this should be lower-case and with double-backtick,
`TF-M does not compile when a ``uart1`` node`

Comment (mia-ko) [reply doc/nrf/releases_and_maturity/known_issues.rst]: I guess it makes more sense :P

**Review by mia-ko — APPROVED** [review]

**Pull request merged**